talk/owt/sdk/p2p/p2ppeerconnectionchannel.cc

```diff
@@ -304,6 +306,8 @@ void P2PPeerConnectionChannel::SendSignalingMessage(
     const Json::Value& data,
     std::function<void()> on_success,
     std::function<void(std::unique_ptr<Exception>)> on_failure) {
+  if (signaling_sender_ == nullptr)
+    return;
```

Comment:
Add a logging warning here so we don't lose track of these if they happen when not related to a connection closing